manifest.json

```diff
@@ -1,5 +1,5 @@
 {
   "name": "Automated merging",
   "description": "Automatically merge pull-requests.",
-  "ubiquity:listeners": [ "push" ]
+  "ubiquity:listeners": [ "push", "pull_request.labeled", "pull_request.unlabeled" ]
```

Comment:
Watching labeling pulls seems unusual since we never do. We do however, label issues regularly.

Reply:
@0x4007 Sorry I am debugging on that branch right now, will remove them when I am done.